app/assets/stylesheets/mobile/markdown_editor.scss

```diff
@@ -0,0 +1,22 @@
+.md-editor {
+  border: 1px solid $light-grey;
+  border-radius: $btn-border-radius-base;
+  overflow: hidden;
+
+  &.active { border-color: $text-grey; }
+
+  &::after,
+  &::before {
+    clear: both;
+    content: ' ';
+    display: table;
+  }
```

Comment:
I removed these lines and I can't spot any difference for comments, conversations and new status messages.